Dockerfile

```diff
@@ -1,8 +1,8 @@
 # kubectl 1.6.6
-FROM google/cloud-sdk:161.0.0-alpine
+#FROM google/cloud-sdk:161.0.0-alpine
 
 # kubectl 1.7.0
-#FROM google/cloud-sdk:163.0.0-alpine
+FROM google/cloud-sdk:163.0.0-alpine
```

Comment:
they're up to 182 now; any reason to not use that? (I don't know the history of why these are both here)

Reply:
I didn't find a build for 182, so I changed it to 181.
https://hub.docker.com/r/google/cloud-sdk/builds/
I don't know the history of this either.